packages/framer-motion/src/components/AnimatePresence/__tests__/AnimatePresence.test.tsx

```diff
@@ -346,6 +346,39 @@ describe("AnimatePresence", () => {
         return await expect(promise).resolves.toBe(3)
     })
 
+    test("Can cycle through multiple components with mode === 'popLayout' and dom", async () => {
```

Comment:
I'm not 100% on what this test is trying to achieve.

Is it more expected based on the changes that we should check that `style` blocks are being correctly added to the `testDom` elements?